### PR TITLE
9.6.0 Release

### PR DESCRIPTION
**Pull request by jennifer-shehane**
- [x] Changelog
- [x] multi-origin docs https://github.com/cypress-io/cypress-documentation/pull/4439 
- [x] session api https://github.com/cypress-io/cypress-documentation/pull/4448
- [x] roadmap updates https://github.com/cypress-io/cypress-documentation/pull/4450
- [x] experiment page updates https://github.com/cypress-io/cypress-documentation/pull/4449

Waiting on:
- https://github.com/cypress-io/cypress/pull/18075
- https://github.com/cypress-io/cypress/pull/20949

~Potentially~ We are not waiting on:
- ~https://github.com/cypress-io/cypress/pull/20704~

### CODE REVIEW

File: content/_changelogs/9.6.0.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  - Added a new Cypress command, called `cy.origin()`, which enables running
+    commands within multiple origins from a single test.
+  - When enabled, cross-origin requests will no longer immediately fail, but
+    instead use the `pageLoadTimeout` configuration as the timeout value. This
+    allows the request to be appropriately handled by the `cy.origin()` command
+    and if it is not handled, it's assumed it was an unexpected request that
+    should fail.
+  - When enabled, tests will no longer wait for the page to load before starting
+    the next tests.
+
```
Comment (emilyrohrbough) [inline content/_changelogs/9.6.0.md:37]: @jennifer-shehane do you feel like the last two bullets are too much? I'm feeling yes. People can read these points in the blog post & the doc page.

File: content/_changelogs/9.6.0.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+**Experiments:**
+
+- The Sessions beta feature has been enhanced by the new beta feature which
+  enables Multi-Origin support. To reflect the intent of using these experiments
+  together, the `experimentalSessionSupport` configuration option has been
+  replaced by the `experimentalSessionAndOrigin` configuration option. The
+  behaviors and usage of the `cy.session()` command has not been changed. Please
+  update your configuration to use the new `experimentalSessionAndOrigin` option
+  to continue to opt-in to this beta feature.
```
Comment (jennifer-shehane) [inline content/_changelogs/9.6.0.md:13]: This should just be in the features section, Experiments is not a normal section for changelogs. I really want the origin announcement to be at the top of the changelog, this is not as important to note.

File: content/_changelogs/9.6.0.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+**Features:**
+
+- We are excited to announce a new beta feature to enable Multi-Origin support.
```
Comment (jennifer-shehane) [inline content/_changelogs/9.6.0.md:17]: Beta and experimental are 2 very different things, we shouldn't use the world Beta in this case. https://docs.google.com/document/d/1NdZdTHW0y9WCPbK6gKNZgekFvxYG7CN7F3YK93R9wHg/edit

```suggestion
- We're excited to announce a new experimental feature to enable Multi-Origin support.
```

File: content/_changelogs/9.6.0.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  This feature allows a user to visit multiple domains of different origins
+  within a single test. This enhances the current test experience which limits a
+  user to only run assertions against a single origin in a test due to standard
+  web security features of the browser. This experiment was designed to work
+  side-by-wide with the Sessions beta feature. For more details, see [our blog
+  post] (TO DO). Addressed
```
Comment (jennifer-shehane) [inline content/_changelogs/9.6.0.md:23]:
```suggestion
  post](https://cypress.io/blog/2022/04/25/cypress-9-6-0-easily-test-multi-domain-workflows-with-cy-origin/)). Addressed
```

**Review by jennifer-shehane — COMMENTED** [review]
Not done reviewing, but a couple of notes.

File: content/_changelogs/9.6.0.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  [#17336](https://github.com/cypress-io/cypress/issues/17336) and
+  [#944](https://github.com/cypress-io/cypress/issues/944) -- TO DO.
+  - Added a new configuration option called `experimentalSessionAndOrigin`. This
+    options enabled both Session support and Multi-Origin support.
+  - Added a new Cypress command, called `cy.origin()`, which enables running
+    commands within multiple origins from a single test.
+  - When enabled, cross-origin requests will no longer immediately fail, but
```
Comment (AtofStryker) [inline content/_changelogs/9.6.0.md:30]:
```suggestion
  - When enabled, cross-origin navigations will no longer immediately fail, but
```

File: content/_changelogs/9.6.0.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  designed to work side-by-side with the [`cy.session()`](/api/commands/session)
+  experimental feature. You can turn on use of this command by setting the
+  [`experimentalSessionAndOrigin`](guides/references/experiments) configuration
+  option to `true`. For more details, read
+  [our blog post](https://cypress.io/blog/2022/04/25/cypress-9-6-0-easily-test-multi-domain-workflows-with-cy-origin/).
+  Addressed [#17336](https://github.com/cypress-io/cypress/issues/17336) and
+  [#944](https://github.com/cypress-io/cypress/issues/944).
+  - Added a new configuration option called `experimentalSessionAndOrigin`. This
+    option enables support for [`cy.session()`](/api/commands/session) and
+    [`cy.origin()`](/api/commands/origin). The `experimentalSessionSupport`
```
Comment (mschile) [inline content/_changelogs/9.6.0.md:19]: Suggest making the removal of the `experimentalSessionSupport` its own bullet point.

File: content/_changelogs/9.6.0.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    configuration option has been removed and replaced by the
+    `experimentalSessionAndOrigin` configuration option. Please update your
+    configuration to use the new `experimentalSessionAndOrigin` option to
+    continue to opt-in to this experimental feature.
+  - When `experimentalSessionAndOrigin` is enabled, cross-origin requests will
```
Comment (mschile) [inline content/_changelogs/9.6.0.md:24]: We should use the same phrasing as in the origin docs: https://github.com/cypress-io/cypress-documentation/pull/4439/files#diff-6eb6403162ee482d3900b46958f5ccd5597b0a53f4fd30045a4a6ef87197057fR34

File: content/guides/references/roadmap.md [hunk 8]
```diff
@@ -15,6 +15,6 @@ Cypress is headed now.
 | _Alpha_            | **Component Testing**           | [#5922](https://github.com/cypress-io/cypress/issues/5922) | [#14479](https://github.com/cypress-io/cypress/pull/14479) | [v7.0.0](/guides/references/changelog#7-0-0) |
 | _Experimental_     | **Cypress Studio**              | [#73](https://github.com/cypress-io/cypress/issues/73)     | [#9542](https://github.com/cypress-io/cypress/pull/9542)   | [v6.3.0](/guides/references/changelog#6-3-0) |
 | _Experimental_     | **Session API**                 | [#8301](https://github.com/cypress-io/cypress/issues/8301) | [#8765](https://github.com/cypress-io/cypress/pull/8765)   | [v8.2.0](/guides/references/changelog#8-2-0) |
+| _Experimental_     | **Visit multiple superdomains** | [#944](https://github.com/cypress-io/cypress/issues/944)   | [#21137](https://github.com/cypress-io/cypress/pull/21137) | [v9.6.0](/guides/references/changelog#9-6-0) |
```
Comment (emilyrohrbough) [inline content/guides/references/roadmap.md:18]:
```suggestion
| _Experimental_     | **Visit multiple origins** | [#944](https://github.com/cypress-io/cypress/issues/944)   | [#21137](https://github.com/cypress-io/cypress/pull/21137) | [v9.6.0](/guides/references/changelog#9-6-0) |
```

**Review by mjhenkes — APPROVED** [review]

File: content/_changelogs/9.6.0.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+  - The `experimentalSessionSupport` configuration option has been removed and
+    replaced by the `experimentalSessionAndOrigin` configuration option. Please
+    update your configuration to use the new `experimentalSessionAndOrigin`
+    option to continue to opt-in to this experimental feature.
+  - When `experimentalSessionAndOrigin` is enabled, cross-origin requests will
+    no longer immediately fail, but instead, time out based on the
+    [`pageLoadTimeout`](/guides/references/configuration#Timeouts)
+    configuration. This allows the request to be appropriately handled by the
+    [`cy.origin()`](/api/commands/origin) command and if it isn't handled, it's
+    assumed it was an unexpected request that should fail.
```
Comment (mschile) [inline content/_changelogs/9.6.0.md:29]: I don't think this explanation is needed in the changelog.

```suggestion
    configuration.
```

File: content/_changelogs/9.6.0.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  - When `experimentalSessionAndOrigin` is enabled, Cypress will no longer
+    prevent running the next command in a hook if a page is still loading as a
+    result of a previously run command in a previous hook.
```
Comment (mschile) [inline content/_changelogs/9.6.0.md:32]: I think this working is kind of hard to understand. How about:
```suggestion
  - When `experimentalSessionAndOrigin` is enabled, Cypress will no longer
     wait on page loads between hooks or before moving on to the next test.
```

File: content/_changelogs/9.6.0.md (hunk 9 above)
Comment (emilyrohrbough) [reply content/_changelogs/9.6.0.md:29]: Typically in the changelog we provide the previous behavior on why a change was needed or what it improving. I think standalone calling this change out doesn't make sense why the change was needed. What if I slim to ```suggestion
    configuration. This allows the request to be appropriately handled by the
    [`cy.origin()`](/api/commands/origin) command.
```

File: content/_changelogs/9.6.0.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    update your configuration to use the new `experimentalSessionAndOrigin`
+    option to continue to opt-in to this experimental feature.
+  - When `experimentalSessionAndOrigin` is enabled, cross-origin requests will
+    no longer immediately fail, but instead, time out based on the
+    [`pageLoadTimeout`](/guides/references/configuration#Timeouts)
+    configuration. This allows the request to be appropriately handled by the
+    [`cy.origin()`](/api/commands/origin) command and if it isn't handled, it's
+    assumed it was an unexpected request that should fail.
+  - When `experimentalSessionAndOrigin` is enabled, Cypress will no longer
+     wait on page loads between hooks or before moving on to the next test.
```
Comment (emilyrohrbough) [inline content/_changelogs/9.6.0.md:31]:
```suggestion
  - When `experimentalSessionAndOrigin` is enabled, Cypress will no longer
     wait on page loads between hooks before moving on to the next test.
```

**Review by mschile — APPROVED** [review]

**Review by debrisapron — APPROVED** [review]

**Pull request merged**